api-client/src/endpoints/submitHelpForm.ts

```diff
@@ -1,6 +1,6 @@
 import OfferModel from '../models/OfferModel'
 
-export const MAX_COMMENT_LENGTH = 200
+export const MALTE_HELP_FORM_MAX_COMMENT_LENGTH = 200
```

Comment:
This number was implicated by the design @hauf-toni made.
But I think I also remember that the comment field was initially not requested by malteser. We added it to help adding some context to the help request. This could be the reason why the length is so short. Maybe we can increase it to 400 characters?

Reply:
Increased to 400